cfme/containers/provider/openshift.py

```diff
@@ -170,6 +181,9 @@ def view_value_mapping(self):
             mapping['proxy'] = None
             mapping['advanced'] = None
 
+        if self.virtualization:
```

Comment:
This needs to be version picked into the code blocks above, I'm thinking:

```
if not self.appliance.is_downstream:  # no is_upstream method
    mapping['virtualization'] = self.virtualization
elif self.appliance.version>= '5.9':
    <existing block of mapping for 5.9>
else:
    <existing block of mapping to None>
    mapping['virtualization'] = self.virtualization

Reply:
I don't think its needed, KubeVirt is upstream only for now
Added just `is_downstream` check with `if self.virtualization`

Reply:
So here's the use-case that I'm thinking about when I consider changes like this - The yaml restricts where the provider is collected via the 'since_version', which means if you're running tests the provider won't be collected when you're testing downstream, and there's no problem.

Outside of running tests, the yaml 'since_version' field doesn't block me from getting a provider crud object and trying to configure it against my appliance, which could be any version.  If I get the crud object for an openshift provider, the blocks above make sure that when I try to fill in the UI form for this provider, only the configuration options relevant for that appliance version are returned. This means that a `provider.create()` call won't attempt to fill widgets that don't exist, and will only fill the form values available for that provider type on that appliance version. The provider may still fail to configure correctly, but it will fail because of CFME validation or endpoint incompatibility - not with an AttributeError or NoSuchElementException type of error.

A separate `if` statement is just fine here, my suggestion was just that it may read better by incorporating it into the existing `if/else` structure.